docs/element-definitions.md

```diff
@@ -18,30 +18,24 @@ The Base API Element contains four properties: `element`, `meta`, `attributes`,
 
     The `element` property defines the name of element. It MUST be a string that references an element, which SHOULD be defined.
 
-- `meta` (enum)
+- `meta`
 
-    The `meta` property is a reserved object for Refract-specific values. When `meta` is an object, it MAY contain elements itself. The element definition SHOULD be used when interacting with `meta` and its properties and values.
+    The `meta` property is a reserved object for Refract-specific values that MAY contain elements itself. The element definition SHOULD be used when interacting with `meta` and its properties and values.
 
-    - Members
-        - (object)
-            - `id` - Unique Identifier, MUST be unique throughout the document
-            - `ref` (Element Pointer) - Pointer to referenced element or type
-            - `classes` (array[string]) - Array of classifications for given element
-            - `title` (string) - Human-readable title of element
-            - `description` (string) - Human-readable description of element
-            - `links` (array[Link Element]) - Meta links for a given element
-        - (array[Member Element])
+    - `id` ([String Element][]) - Unique Identifier, MUST be unique throughout the document
+    - `ref` (Ref Element) - Pointer to referenced element or type
+    - `classes` (Array Element) - Array of classifications for given element
+        - `content` (array[[String Element][]], fixed-type)
+    - `title` ([String Element][]) - Human-readable title of element
+    - `description` ([String Element][]) - Human-readable description of element
+    - `links` (Array Element[Link Element]) - Meta links for a given element
```

Comment:
Basically, this is the template feature in MSON. But for this to make sense, we need to edit the refract spec too. If we don't want to use this feature, we can do how it was done in https://github.com/apiaryio/api-elements/pull/22/files#diff-0fd90a60122f956cc63014ff8125a47cR27